service/src/main/kotlin/app/cash/backfila/service/runner/statemachine/BatchAwaiter.kt

```diff
@@ -205,6 +209,16 @@ class BatchAwaiter(
     }
   }
 
+  fun saveProgress(dbRunPartition: DbRunPartition) {
```

Comment:
updateProgress? It doesn't actually save.

Reply:
I'm not thinking of this name from a hibernate terminology

Reply:
But it doesn't actually save to the Db. This confused me when I read it. Took me a while to understand.

"bookmarkProgress"?

Reply:
I don't mind calling it update

Reply:
made it "update"